fixture-monkey-tests/java-tests/src/test/java/com/navercorp/fixturemonkey/tests/java/BeanArbitraryIntrospectorTest.java

```diff
@@ -47,4 +48,11 @@ void setPropertyPostCondition() {
 
 		then(actual).hasSizeGreaterThan(5);
 	}
+
+	@RepeatedTest(TEST_COUNT)
+	void objectGenerationShouldWorkWithoutSetter() {
```

Comment:
It is much better to write a test case that describes the simplest possible version of the issue.

In this case, it would be `StringObject`.